packages/editor/src/helpers/blockConstant.ts

```diff
@@ -0,0 +1,10 @@
+import * as BLOCK from './block'
+
+export const TRANS_TYPE_LIST = [
```

Comment:
I plan to remove helpers progressively...

how about keeping constant inside where they need to be?